File: JAVA_COURSE/JAVA/06_FirstProgram/src/firstprogram/Second.java
package firstprogram;
import static java.lang.System.*;
public class Second {
    private FirstProgram program=new FirstProgram();
   static {out.println("static second объекта" );}
   public Second () 
    {out.println("second объекта"+ program  );}
   public static void printtt() {
       {System.out.println("printtt second объекта"  );return;}
   }
}

File: JAVA_COURSE/KOTLIN/06_FirstProgram/src/kotlinfirstprogram/Second.kt
package kotlinfirstprogram

import java.lang.System.out

class Second {
    private val program = FirstProgram()

    init {
        out.println("second объекта$program")
    }

    companion object {
        init {
            out.println("static second объекта")
        }

        fun printtt() {
            run {
                println("printtt second объекта")
                return
            }
        }
    }
}
